src/dataRepository.cpp

```diff
@@ -0,0 +1,172 @@
+//
+// Created by rita on 17-03-2023.
+//
+
+#include <iostream>
+#include "dataRepository.h"
+
+using namespace std;
```

Comment:
Shouldn't we avoid using namespaces in auxiliary files, so we don't affect namespace usage in other people's code if they use this? Specially since it's not explicit that we do this, and you end up using "std::" in here the same.
(I know no one is gonna use it, but gotta have good practices ig)

Reply:
I changed it up to be more consistent, using std namespace only in source code files and cleaning up the extra std:: references